prov/util/src/util_attr.c

```diff
@@ -220,6 +220,7 @@ int ofix_getinfo(uint32_t version, const char *node, const char *service,
 					fi, alter_base_info, &temp);
 			if (ret)
 				goto err3;
+			ofi_alter_info(temp, hints);
```

Comment:
This patch applies for the rxd/rxm providers which are disabled currently. I think it is ok to merge this after rc2.